pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -82,26 +82,29 @@
 
 You can use `Trainer(num_sanity_val_steps=0)` to skip the sanity check.
 
-# Testing loop
+# Validation or Testing loop
 
-To ensure you don't accidentally use test data to guide training decisions Lightning
- makes running the test set deliberate.
+To ensure you don't accidentally use validation or test data to guide training decisions Lightning
+ makes running the validation or test set deliberate.
```

Comment:
this doesn't make sense.... validation should run within training. 
This change here is wrong. @Borda  @jeremyjordan